projects/prison-custody-status-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/event/Disposal.kt

```diff
@@ -37,7 +43,12 @@ class Disposal(
 
     @Column(updatable = false, columnDefinition = "NUMBER")
     val softDeleted: Boolean = false,
-)
+) {
+    fun isLongerThan20Months(): Boolean? {
```

Comment:
I made it nullable because both `lengthInDays` and `notionalEndDate` are nullable - so there is a scenario where the sentence being longer than 20 months doesn't really make sense, or is unknown.  E.g. for a fine.

We only care if it's true in this service, so I can have it return false for the null case then make the method non-nullable if you prefer?  We'd just have to be careful if we copy the method for another service.

What's the criteria for choosing an extension function over a normal one?  Given we control the source class, and it's not a data class, I thought it made sense to do it the normal way.  Happy to move it, just keen to know which to choose in future.